utils/directories.cc

```diff
@@ -76,14 +76,14 @@ directories::directories(bool developer_mode)
         : _developer_mode(developer_mode)
 { }
 
-future<> directories::create_and_verify(directories::set dir_set) {
+future<> directories::create_and_verify(directories::set dir_set, recursive r) {
```

Comment:
`r` is not descriptive enough for this parameter.
We use single-letter names for variables that are used in smaller scope, or some we use by convention (or used to use) like `t` for table and `s` for schema.
But let's not add more.

Reply:
About the patch title and comment.
nit: needs a space between data and directory.
If the patch can be applied as stand alone without breaking anything, it's ok to keep it separate and change the note about squashing, just to mention the next step, to provide the motivation for the patch.

Reply:
Ack.